tests/sxg_buffer_test.cc

```diff
@@ -14,20 +14,16 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include "libsxg/sxg_buffer.h"
-
 #include <string>
 
 #include "gtest/gtest.h"
 #include "libsxg/internal/sxg_buffer.h"
+#include "libsxg/sxg_buffer.h"
```

Comment:
You should not move this `#include`.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Reverted.